backtrace-library/src/main/cpp/include/breakpad-backend.h

```diff
@@ -0,0 +1,48 @@
+#ifndef BACKTRACE_ANDROID_BREAKPAD_BACKEND_H
+#define BACKTRACE_ANDROID_BREAKPAD_BACKEND_H
+
+#include "exception_handler.h"
+#include "common/linux/http_upload.h"
+#include "cacert.h"
+#include "client-side-unwinding.h"
+#include <jni.h>
+#include <mutex>
+#include <atomic>
+#include <unordered_map>
+#include <string>
+
+static constexpr size_t MAX_UPLOADS_PER_RUN = 3;
+
+extern std::string thread_id;
+extern std::atomic_bool initialized;
+extern std::mutex attribute_synchronization;
+
+static google_breakpad::ExceptionHandler* eh;
```

Comment:
We should not declare static variables in headers, ever.

Reply:
Ah sorry, will move these to implementations to avoid having multiple copies of the static variables